platforms/core-configuration/model-core/src/test/groovy/org/gradle/internal/reflect/ClassInspectorTest.groovy

```diff
@@ -366,12 +366,12 @@ class ClassInspectorTest extends Specification {
         expect:
         ClassInspector.inspect(Object).superTypes.empty
         ClassInspector.inspect(Serializable).superTypes.empty
-        ClassInspector.inspect(List).superTypes.toList() == [Collection, Iterable]
+        ClassInspector.inspect(Set).superTypes.toList() == [Collection, Iterable]
```

Comment:
What's this about?

Reply:
In Java 21, List implements SequencedCollection as well. Using Set allows the test to be consistent across versions.